config/schemas/skill.json

```diff
@@ -18,17 +18,20 @@
                     "description": "localizable description"
                 },
                 "effects": {
-                    "type": "array",
-                    "items": {
+                    "type": "object",
+                    "additionalProperties" : {
                         "$ref" : "vcmi:bonus"
                     }
                 }
             }
         },
 
-        "required" : ["basic", "advanced", "expert"],
+        "required" : ["base", "basic", "advanced", "expert"],
 
         "properties": {
+            "base":{
+                "$ref" : "#/definitions/skillBonus"

```

Comment:
"base" itself do not have to be valid "skillBonus".
Example from spell schema:
` "base":{`
`	"type": "object",`
`	"description": "will be merged with all levels",`
`	"additionalProperties": true`
`},`

Reply:
fixed.